doc/modules/manifold.rst

```diff
@@ -502,6 +502,7 @@ three parameters that control the optimization of t-SNE:
 * early exaggeration factor
 * learning rate
 * maximum number of iterations
+* angle
```

Comment:
Maybe `optimization of t-SNE` should be rephrased `optimization of t-SNE and therefore possibly the quality of the resulting embedding`